config/initializers/as_inflections.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.singular(/(base)s$/i, '\1') # Override rails default: "bases".singularize => "base" not "basis"
+end
```

Comment:
@bdunne Gah I wish I had seen this earlier, but this is all in the wrong place.  Our inflections live [here](https://github.com/ManageIQ/manageiq/blob/master/config/application.rb#L109) now because of timing issues with respect to Settings.

I wish we could do it a nicer way, so I am all ears for a better implementation of that.